.github/workflows/docker.yaml

```diff
@@ -0,0 +1,62 @@
+name: Publish Docker
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image (optional)'
+        required: false
+  create:
+    tags:
+      - '*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  attestations: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
To create multiarch images shouldn't it run on different OSes too? Never done that before

Reply:
No, its not necessary.

Reply:
... or at least this is how I got it working locally and in the original project.